charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml

```diff
@@ -0,0 +1,55 @@
+#### HPE Dynamic Provisioner ###
+---
+apiVersion: extensions/v1beta1
```

Comment:
can we please update to `apps/v1` for k8s v1.16 capability support, thanks.

Reply:
done..